implementation/domain/web/jbb-frontend-web/src/main/webapp/WEB-INF/templates/acp/system.html

```diff
@@ -10,6 +10,8 @@
         <dd>- Monitoring measures and calculate statistics on real operation of a board depending on
             the usage by users
         </dd>
+        <dt>Session</dt>
```

Comment:
Session**s**. And move it to 1st position (not 3rd) just like it is visible in left menu